src/js/zoneUtils.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 exports = module.exports = {
-    STATUS_NOT_STARTED: 1,
-    STATUS_FINISHED: 2,
+    STATUS_NOT_STARTED: 0,
```

Comment:
Yeah, I thought of that. Though I have a feeling `STATUS_IN_PROGRESS` is coming back, so I went with the numbers.

Thinking of this, I'm going to rework how I calculate the progress to be a bit more resilient.